compiler/il/OMRILOps.hpp

```diff
@@ -702,11 +702,7 @@ class ILOpCode
       {
       switch(type)
          {
-         case TR::Int8:     return TR::buadd;
-         case TR::Int16:    return TR::cadd;
-         case TR::Int32:    return TR::iuadd;
-         case TR::Int64:    return TR::luadd;
```

Comment:
I think these cases should remain and just map to badd, sadd, iadd, ladd since the unsigned and signed cases are the same. The API could be deprecated and replaced, but there is a valid opcode for an unsigned add which this should return IMO (it is just the same a signed add.

Reply:
I deleted them with the assumption that the function was not in used. Have fixed the problem in the commit 1e7b727 .